<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易属性连接编辑器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="connect_attr_le"/>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>被连接属性：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="ls_type_nodes_le">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>按类型列出节点：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>连接属性：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2" colspan="4">
    <widget class="QTextEdit" name="connected_obj_te"/>
   </item>
   <item row="2" column="10">
    <widget class="QPushButton" name="connected_attr_pb">
     <property name="text">
      <string>↑通道栏选取</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="connect_attr_pb">
     <property name="text">
      <string>↑通道栏选取</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="12">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="ls_type_nodes_pb">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>ls↓</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QLineEdit" name="connected_attr_le"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="connect_pb">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>正向连接</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>按选中填入节点：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="connected_obj_pb">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>选择物体↓</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="sel_node_mesh_pb">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>sl -&gt; mesh</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="reverse_connect_pb">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>反向连接</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
